util/mkpasswd.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2015 Apcera Inc. All rights reserved.
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"flag"
+	"fmt"
+	"log"
+	"math/big"
+
+	"golang.org/x/crypto/bcrypt"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+func usage() {
+	log.Fatalf("Usage: mkpasswd -p <stdin password> \n")
+}
+
+const (
+	// Make sure password reasonably long to generate enough entropy
+	PasswordLength = 22
+	// Make cost reasonably expensive, min is 4, max is 31
```

Comment:
This explains the constraints but does _not_ address the raised issue of describing how the value was chosen.  Why 11?  As documentation for future maintainers, what did you consider and when should this be increased?

```
// Common advice from the past couple of years suggests that 10 should be sufficient.  Up that a little, to 11.
// Feel free to raise this higher if this value from 2015 is no longer appropriate.
```